src/general/OptionsMenu.cs

```diff
@@ -1002,6 +1002,11 @@ private void OnTutorialsOnNewGameToggled(bool pressed)
     private void OnCheatsToggled(bool pressed)
     {
         Settings.Instance.CheatsEnabled.Value = pressed;
+        if (!pressed)
+        {
+            CheatMenu.Instance.DisableAllCheats();
```

Comment:
Why does this just blindly call into the cheat menu here?
I think there is a need to have an autoload CheatManager singleton that could be used instead.